_data/sidebars/scos_dev_sidebar.yml

```diff
@@ -3886,6 +3886,8 @@ entries:
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-warehouse-data.html
           - title: HowTo - Import Packaging Units
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-packaging-units.html
+          - title: 'HowTo: Manage merchant custom prices for products via data import'
```

Comment:
We should not label it as Merchant custom prices but more "HowTo: Manage business unit specific prices via data import"